app/pages/resource/reservation-form/ReservationFormContainer.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import ReservationForm from './ReservationForm';
+
+const resourceSelector = (state, props) => props.resource;
```

Comment:
Maybe it's a good idea to keep the selectors in a different file. Maybe that way we could encourage the reusability of the most common ones. 

If we wouldn't have render logic in the Container where would we have the kind of logic we currently define in the containers? 

This Container exports a connected Form. I encountered a difficulty using this pattern at the time of testing it. As I wanted to test that the ReservationForm was being rendered with a correct initialValue, but it's difficult to test a connected item in tests.

About the actions, if I recall correctly we usually just bind the dispatcher to the actions and pass them to the connect function. If it's just that I guess we don't need a file to define the actions. Perhaps we could export the mapDispatchToProps function we have used in order to test it. 

So I guess I'ld vote for 2 or 3, depending on the reasons of not having rendering logic on the container.